File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/Nuro/NuroRES1001U00ReservationScheduleConditionListItemInfo.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.Nuro
{
    [Serializable]
	public class NuroRES1001U00ReservationScheduleConditionListItemInfo
	{
		private String _amStartTime;
		private String _amEndTime;
		private String _pmStartTime;
		private String _pmEndTime;

		public String AmStartTime
		{
			get { return this._amStartTime; }
			set { this._amStartTime = value; }
		}

		public String AmEndTime
		{
			get { return this._amEndTime; }
			set { this._amEndTime = value; }
		}

		public String PmStartTime
		{
			get { return this._pmStartTime; }
			set { this._pmStartTime = value; }
		}

		public String PmEndTime
		{
			get { return this._pmEndTime; }
			set { this._pmEndTime = value; }
		}

		public NuroRES1001U00ReservationScheduleConditionListItemInfo() { }

		public NuroRES1001U00ReservationScheduleConditionListItemInfo(String amStartTime, String amEndTime, String pmStartTime, String pmEndTime)
		{
			this._amStartTime = amStartTime;
			this._amEndTime = amEndTime;
			this._pmStartTime = pmStartTime;
			this._pmEndTime = pmEndTime;
		}

	}
}

File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/nuro/NuroRES1001U00ReservationScheduleConditionListItemInfo.java
package nta.med.data.model.ihis.nuro;

public class NuroRES1001U00ReservationScheduleConditionListItemInfo {
	private String amStartTime;
	private String amEndTime;
	private String pmStartTime;
	private String pmEndTime;
	
	public NuroRES1001U00ReservationScheduleConditionListItemInfo(
			String amStartTime, String amEndTime, String pmStartTime,
			String pmEndTime) {
		this.amStartTime = amStartTime;
		this.amEndTime = amEndTime;
		this.pmStartTime = pmStartTime;
		this.pmEndTime = pmEndTime;
	}

	public String getAmStartTime() {
		return amStartTime;
	}

	public void setAmStartTime(String amStartTime) {
		this.amStartTime = amStartTime;
	}

	public String getAmEndTime() {
		return amEndTime;
	}

	public void setAmEndTime(String amEndTime) {
		this.amEndTime = amEndTime;
	}

	public String getPmStartTime() {
		return pmStartTime;
	}

	public void setPmStartTime(String pmStartTime) {
		this.pmStartTime = pmStartTime;
	}

	public String getPmEndTime() {
		return pmEndTime;
	}

	public void setPmEndTime(String pmEndTime) {
		this.pmEndTime = pmEndTime;
	}
}
